Index: Makefile.am
===================================================================
--- Makefile.am	(revision 7202)
+++ Makefile.am	(working copy)
@@ -9,7 +9,6 @@
 docfiles_DATA = README \
 				README.html \
 				ChangeLog \
-				COPYING \
 				THANKS \
 				AUTHORS 
 
--- Makefile.in
+++ Makefile.in
@@ -273,7 +273,6 @@
 docfiles_DATA = README \
 				README.html \
 				ChangeLog \
-				COPYING \
 				THANKS \
 				AUTHORS 
 
