Description: don't install useless COPYING file
Forwarded: not-neded

--- a/Makefile.am
+++ b/Makefile.am
@@ -7,8 +7,6 @@
 docfilesdir = $(docdir)
 docfiles_DATA = README \
 				README.html \
-				ChangeLog \
-				COPYING \
 				THANKS \
 				THANKS.artists \
 				AUTHORS 
--- a/Makefile.in
+++ b/Makefile.in
@@ -298,8 +298,6 @@
 docfilesdir = $(docdir)
 docfiles_DATA = README \
 				README.html \
-				ChangeLog \
-				COPYING \
 				THANKS \
 				THANKS.artists \
 				AUTHORS 
