Index: Makefile
===================================================================
--- Makefile	(revision 6485)
+++ Makefile	(working copy)
@@ -89,7 +89,6 @@
 	done
 	if [[  -n $$(find po -name *.mo) ]]; then \
 		${MAKE} -C po install PREFIX=$(PREFIX) ; \
 	fi
-	cp COPYING "$(DESTDIR)$(PREFIX)/share/gajim/";
 	cp THANKS "$(DESTDIR)$(PREFIX)/share/gajim/";
 	cp AUTHORS "$(DESTDIR)$(PREFIX)/share/gajim/";
 	mkdir -p "$(DESTDIR)$(PREFIX)/share/pixmaps";
Index: src/dialogs.py
===================================================================
--- src/dialogs.py	(revision 6485)
+++ src/dialogs.py	(working copy)
@@ -558,7 +558,7 @@
 		dlg.set_version(gajim.version)
 		s = u'Copyright \xa9 2003-2005 Gajim Team'
 		dlg.set_copyright(s)
-		text = open('../COPYING').read()
+		text = open('/usr/share/common-licenses/GPL-2').read()
 		dlg.set_license(text)
 
 		dlg.set_comments(_('A GTK jabber client'))
