From a3e037dfc2190e6de92e34418a6ab4a2de30d1bd Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Philipp=20H=C3=B6rist?= <philipp@hoerist.com>
Date: Sat, 1 Jun 2024 00:55:52 +0200
Subject: [PATCH] Remove nbxmpp from deps

---
 pyproject.toml | 1 -
 1 file changed, 1 deletion(-)

diff --git a/pyproject.toml b/pyproject.toml
index f0fbbc2c6..58207a0c6 100644
--- a/pyproject.toml
+++ b/pyproject.toml
@@ -29,7 +29,6 @@ dependencies = [
   "css-parser",
   "emoji>=2.6.0",
   "keyring",
-  "nbxmpp>=5.0.4,<6.0.0",
   "omemo-dr>=1.0.0,<2.0.0",
   "packaging",
   "pillow>=9.1.0",
-- 
2.43.0

