Index: Makefile
===================================================================
--- Makefile	(revision 2269)
+++ Makefile	(working copy)
@@ -64,7 +64,6 @@
 		DST=`dirname "$$f"`; \
 		cp "$$f" "$(DESTDIR)$(PREFIX)/share/gajim/$$DST/"; \
 	done
-	cp COPYING "$(DESTDIR)$(PREFIX)/share/gajim/";
 	mkdir -p "$(DESTDIR)$(PREFIX)/share/pixmaps";
 	cp data/pixmaps/gajim.png "$(DESTDIR)$(PREFIX)/share/pixmaps/";
 	cp data/pixmaps/gajim_about.png "$(DESTDIR)$(PREFIX)/share/pixmaps/";
Index: src/dialogs.py
===================================================================
--- src/dialogs.py	(revision 2272)
+++ src/dialogs.py	(working copy)
@@ -391,7 +391,7 @@
 		dlg.set_version(gajim.version)
 		s = u'Copyright \xa9 2003-2005 Gajim Team'
 		dlg.set_copyright(s)
-		text = open('../COPYING').read()
+		text = open('/usr/share/common-licenses/GPL-2').read()
 		dlg.set_license(text)
 
 		dlg.set_comments(_('A GTK jabber client'))
